Detectors/TPC/calibration/doc/CalibdEdx.md

```diff
@@ -1,39 +1,41 @@
 <!-- doxy
-\page refTPCcalibrationCalibdEdx dEdx Calibration
+\page refTPCcalibrationCalibdEdx Residual dEdx Calibration
 /doxy -->
 
-# dEdx Calibration
+# Residual dEdx Calibration
 
-The workflow `o2-tpc-miptrack-filter` is supposed to run in the EPNs, and will select only the track inside the defined cuts, the selected track in the streamed with the name `TPC/MIPS/0`. The cut values can be changed with these CLI options:
+The workflow `o2-tpc-miptrack-filter` will run on the EPNs. It selects only the tracks inside the defined cuts and stream them with the name `TPC/MIPS/0`. The cut values can be changed with these CLI options:
 
 ```
---min-momentum (= 0.4)
---max-momentum (= 0.6)
+--min-momentum (= 0.3)
+--max-momentum (= 0.7)
 --min-clusters (= 60)       Minimum number of cluster in a track.
 ```
 
-The workflow `o2-tpc-calibratordedx` should run in an aggregation node. It will fill dEdx histograms using the data sended by the `o2-tpc-miptrack-filter`, and will compute corrections for the dE/dx values for every time slot.
+The workflow `o2-tpc-calibratordedx` should run on an aggregation node. It fills dEdx histograms using the data sended by the `o2-tpc-miptrack-filter`, and computes corrections for the dE/dx values for every time slot.
```

Comment:
```suggestion
The workflow `o2-tpc-calibratordedx` should run on an aggregation node. It fills dEdx histograms using the data sent by the `o2-tpc-miptrack-filter`, and computes corrections for the dE/dx values for every time slot.
```

Reply:
I'm committing the suggestions for now, after I will squash everything